cmd/server-main.go

```diff
@@ -134,13 +134,12 @@ func serverMain(ctx *cli.Context) {
 
 	// Get "json" flag from command line argument and
 	// enable json and quite modes if jason flag is turned on.
-	jsonFlag := ctx.Bool("json") || ctx.GlobalBool("json")
-	if jsonFlag {
+	if ctx.IsSet("json") || ctx.GlobalIsSet("json") {
```

Comment:
Just change `Bool` to `IsSet` and keep `jsonFlag`.  Later it will be used for command line argument comparison.